tests/aws/services/lambda_/test_lambda_api.snapshot.json

```diff
@@ -12521,6 +12521,150 @@
       }
     }
   },
+  "tests/aws/services/lambda_/test_lambda_api.py::TestLambdaSnapStart::test_snapstart_lifecycle[java21]": {
+    "recorded-date": "02-05-2023, 18:15:11",
```

Comment:
Snapshots always need to be generated automatically as explained under [Parity Testing](https://docs.localstack.cloud/contributing/parity-testing/). They must never be edited manually. Hence, the date from spring cannot be correct here.

This great blog post is the best introduction: https://blog.localstack.cloud/2022-08-04-parity-explained/